ramls/routing-list.raml

```diff
@@ -0,0 +1,81 @@
+#%RAML 1.0
+title: "mod-orders-storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1.0
+
+documentation:
+  - title: Routing List
+    content: <b>This module implements the CRUD interface. This API is intended for internal use only. </b>
```

Comment:
We don't need to specify here "This API is intended for internal use only."
Also need to add info that this API is for routing lists